docker-images/kafka-based/kafka/exporter-scripts/kafka_exporter_run.sh

```diff
@@ -30,6 +30,10 @@ if [ "$KAFKA_EXPORTER_ENABLE_SARAMA" = "true" ]; then
     saramaenable="--log.enable-sarama"
 fi
 
+if [ "KAFKA_EXPORTER_OFFSET_SHOW_ALL" != "false" ]; then
+    allgroups="--offset.show-all"
+fi
```

Comment:
I made a positive condition at first, then I thought that we have opposite condition here. But it seems positive condition is better :) It will be fixed 👍